compiler/luci/pass/src/SubstituteTransposeToReshapePass.cpp

```diff
@@ -92,6 +92,8 @@ bool substitute_transpose_to_reshape(luci::CircleTranspose *node)
   new_const_node->name(name + "/Reshape/shape");
   luci::add_origin(new_const_node, luci::get_origin(node));
 
+  luci::copy_quantparam(node, new_reshape_node);
```

Comment:
The current master does not copy qparam, which causes wrong conversion in the quantized model.